ios/Classes/SITFSDKPlugin.m

```diff
@@ -590,4 +592,28 @@ - (void) handleGeofenceCallbacksRequested :(FlutterMethodCall*)call
     result(@"SUCCESS");
 }
 
+- (void)handleConfigureUserHelper:(FlutterMethodCall*)call result:(FlutterResult)result {
+    BOOL enabled = [call.arguments[@"enabled"] boolValue];
+    NSDictionary *colorScheme = call.arguments[@"colorScheme"];
+    
+    [[SITUserHelperManager sharedInstance] autoManage:enabled];
+    
+    if (colorScheme) {
+        NSString *primaryColor = colorScheme[@"primaryColor"];
+        NSString *secondaryColor = colorScheme[@"secondaryColor"];
+        
```

Comment:
Non se poderia facer un [SITUserHelperColorScheme fromDictionary] ou incluso un [[SITUserHelperManager sharedInstance] autoManageFromDictionary, para non repetir esta loxica nos distintos plugins

Reply:
Si, pero os SDKs non implementan esto e sinceramente penso que me dá máis choio. Xa o iremos metendo.

Reply:
Se me paso por completo meterlos, que fallo